src/helpers.c

```diff
@@ -528,54 +528,61 @@ void swupd_deinit(void)
 *	- Initialize curl
 *	- Initialize signature checking
 */
-enum swupd_code swupd_init(void)
+enum swupd_code swupd_init(enum swupd_init_config config)
 {
 	int ret = SWUPD_OK;
```

Comment:
you changed `int ret` variables in other functions to be `enum swupd_code`, but you left this one. Did you miss it?

Reply:
yes. fixing it.